Backport from current gnulib/coreutils cvs to hide some symbols that otherwise
can cause build failures when linking statically.

http://bugs.gentoo.org/131674

--- lib/regex_internal.h
+++ lib/regex_internal.h
@@ -122,9 +122,6 @@
 # define __attribute(arg)
 #endif
 
-extern const char __re_error_msgid[] attribute_hidden;
-extern const size_t __re_error_msgid_idx[] attribute_hidden;
-
 typedef __re_idx_t Idx;
 
 /* Special return value for failure to match.  */
--- lib/regcomp.c
+++ lib/regcomp.c
@@ -125,7 +125,7 @@
    POSIX doesn't require that we do anything for REG_NOERROR,
    but why not be nice?  */
 
-const char __re_error_msgid[] attribute_hidden =
+static const char __re_error_msgid[] =
   {
 #define REG_NOERROR_IDX	0
     gettext_noop ("Success")	/* REG_NOERROR */
@@ -179,7 +179,7 @@
     gettext_noop ("Unmatched ) or \\)") /* REG_ERPAREN */
   };
 
-const size_t __re_error_msgid_idx[] attribute_hidden =
+static const size_t __re_error_msgid_idx[] =
   {
     REG_NOERROR_IDX,
     REG_NOMATCH_IDX,
